frontend/src/components/EmailPopUp.vue

```diff
@@ -0,0 +1,139 @@
+<template>
+  <q-card class="bg-white text-white event-popup">
+    <q-bar class="email-bar">
+      <div class="text-h6">New Message</div>
+      <q-space />
+      <q-btn dense flat icon="close" v-close-popup></q-btn>
+    </q-bar>
+    <q-card-section>
+      <!-- Email form  -->
+      <q-form @submit="sendEmail" class="q-gutter-md">
+        <q-select
+          v-model="selectedEmails"
+          label="To"
+          :options="filterOptions"
+          multiple
+          use-chips
+          use-input
+          input-debounce="0"
+          @new-value="createRecipient"
+          @filter="filterEmails"
+          stack-label
+          lazy-rules
+          :rules="[
+            val =>
+              (val && val.length > 0) || 'Please enter at least one recipient'
+          ]"
+        />
+        <q-input
+          v-model="subject"
+          label="Subject"
+          lazy-rules
+          :rules="[
+            val =>
+              (val && val.length > 0) || 'Please enter a subject'
+          ]"
+        />
+        <q-input v-model="msg" type="textarea" />
```

Comment:
Add a label here (e.g. `label="Body"`) to be consistent with the input for the subject?